packages/shared/src/components/post/PostComments.tsx

```diff
@@ -137,6 +139,7 @@ export function PostComments({
           postScoutId={post.scout?.id}
           appendTooltipTo={modalParentSelector ?? (() => container?.current)}
           permissionNotificationCommentId={permissionNotificationCommentId}
+          joinNotificationCommentId={joinNotificationCommentId}
```

Comment:
Not happy with this props drilling, but the same pattern was used before so did not wanna make to drastic changes. It would be cool to have some local context here for comments and banners. Something like this could be useful https://beta.kickass.codes/docs/react/factories/create-context-provider

Reply:
We do have a react query hook somewhere that can act in these cases?
Or even the one we used for the animations maybe if it's per sessions only

Reply:
Yeah, also valid options, but I would prefer doing that as backlog item since we have other MVP related stuff to do?

Reply:
Yep 100% fine with me. 
Can you maybe add it to backlog for us?

Reply:
Added https://dailydotdev.atlassian.net/browse/WT-1596?atlOrigin=eyJpIjoiZDliNDAyYmFjOWQwNGI1OWJlNTg4MDU0ZTk2YWViMWQiLCJwIjoiaiJ9